services/src/projects/project.rs

```diff
@@ -0,0 +1,178 @@
+use uuid::Uuid;
+use crate::workflows::Workflow;
+use crate::users::user::{UserIdentification, UserInput};
+use chrono::{DateTime, Utc};
+use serde::{Deserialize, Serialize};
+use crate::error::Error;
+use crate::error;
+use snafu::ensure;
+
+#[derive(Debug, PartialEq, Eq, Serialize, Deserialize, Clone, Copy, Hash)]
+pub struct ProjectId {
+    id: Uuid
+}
+
+impl ProjectId {
+    #[allow(clippy::new_without_default)]
+    pub fn new() -> Self {
+        Self {
+            id: Uuid::new_v4()
+        }
+    }
+}
+
+#[derive(Debug, PartialEq, Eq, Serialize, Deserialize, Clone)]
+pub struct Project {
+    pub id: ProjectId,
+    pub version: usize,
+    pub changed: DateTime<Utc>,
+    pub name: String,
+    pub description: String,
+    pub layers: Vec<Layer>,
+    //TODO: stref
```

Comment:
what do we need/want to store for a project? The current/latest view of a user? Do projects also get limiting bounds? So that the may user may not zoom out, or at least there are no queries issued outside of the bounds.

Reply:
Yeah, so there seem to be requirements for such bounds for certain projects.
Either you include the option to have bounds or we could postpone this decision.

Reply:
ok so one strect for view and one for bounds

Reply:
5d17f4f97772880e419f6f3a81ea4d2bb6ff1530